src/pages/Teams/ListTeams/ListTeams.jsx

```diff
@@ -0,0 +1,124 @@
+import React, { useEffect } from 'react';
+import {
+  Button,
+  Stack,
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableHead,
+  TableRow,
+  Tooltip,
+  Typography,
+} from '@mui/material';
+import GroupAddIcon from '@mui/icons-material/GroupAdd';
+import PersonAddIcon from '@mui/icons-material/PersonAdd';
+import EditIcon from '@mui/icons-material/Edit';
+import DeleteIcon from '@mui/icons-material/Delete';
+import { useNavigate } from 'react-router-dom';
+import { useDispatch, useSelector } from 'react-redux';
+import { getTeams } from '../../../store/actions/teams';
+
+function capitalize(text) {
+  return text.charAt(0).toUpperCase() + text.slice(1);
+}
+
+const ListTeams = () => {
+  const dispatch = useDispatch();
+  const navigate = useNavigate();
+  const { teams } = useSelector((state) => state.teams);
+
+  useEffect(() => {
+    dispatch(getTeams());
+  }, [dispatch]);
+
+  return (
+    <>
+      <Stack direction="row" justifyContent="space-between" alignItems="flex-start" spacing={2}>
+        <Typography variant="h3" component="h3">
+          List of Teams
+        </Typography>
+        <Button
+          variant="outlined"
+          onClick={() => {
+            navigate('/teams/add');
+          }}
+        >
+          <GroupAddIcon
+            sx={{
+              padding: '0 10px 0 0',
+            }}
+          />
+          Add a team
+        </Button>
```

Comment:
1.  Use `startIcon` props instead of setting the icon directly in the component
2. I configured MUI Button to automatically use a custom component (ButtonLink) when we call Button so you can use `href="link"` instead of navigating with react router hook

```suggestion
        <Button
          variant="outlined"
          href="/teams/add
          startIcon={<GroupAddIcon />}
        >
          Add a team
        </Button>
```